packages/common/src/env.ts

```diff
@@ -1,6 +1,7 @@
 export type Env = {
   APP_VERSION: string;
   APP_API_BASE_URL: string;
+  APP_API_ACCESS_BRIDGE_URL: string;
```

Comment:
As we discussed the other day, the middleware url will be different per customer, so should probably be an ini file param

Reply:
nm. I see it's in both places.